spring-cloud-skipper-server/src/test/java/org/springframework/cloud/skipper/AbstractIntegrationTest.java

```diff
@@ -33,4 +39,28 @@
 @SpringBootTest
 @DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
 public abstract class AbstractIntegrationTest {
+
+	@Autowired
+	protected ReleaseRepository releaseRepository;
+
+	@Autowired
+	protected ReleaseService releaseService;
+
+	@After
+	public void cleanupReleases() {
+		// Add a sleep for now to give the local deployer a chance to install the app. This
+		// should go away once we introduce spring state machine.
+		try {
+			Thread.sleep(5000);
+			for (Release release : releaseRepository.findAll()) {
+				if (release.getInfo().getStatus().getStatusCode() != StatusCode.DELETED) {
+					releaseService.delete(release.getName());
+				}
+			}
+		}
+		catch (Exception e) {
+			System.out.println("error cleaning up resource in integration test");
```

Comment:
Even for tests, proper logging is better.

Reply:
righto!